The build failures look to be caused by this change:

```
/__w/swift-foundation/swift-foundation/Sources/FoundationEssentials/Predicate/Predicate.swift:59:90: error: static method 'build_evaluate' requires the types 'PredicateExpressions.PredicateRegex' and 'Expression<repeat each Input, Bool>' be equivalent
```

--- a/Sources/FoundationEssentials/Predicate/Predicate.swift
+++ b/Sources/FoundationEssentials/Predicate/Predicate.swift
@@ -56,7 +56,7 @@ extension Predicate {
         
         if let guarded = iterator.next() {
             self.init({ (input: repeat PredicateExpressions.Variable<each Input>) in
-                let condition: PredicateExpressions.Value<Self> = PredicateExpressions.build_Arg(guarded)
+                let condition = PredicateExpressions.Value(guarded)
                 var pieces: any StandardPredicateExpression<Bool> = PredicateExpressions.build_evaluate(condition, repeat each input)
                 
                 while let next = iterator.next() {
@@ -75,7 +75,7 @@ extension Predicate {
         
         if let guarded = iterator.next() {
             self.init({ (input: repeat PredicateExpressions.Variable<each Input>) in
-                let condition: PredicateExpressions.Value<Self> = PredicateExpressions.build_Arg(guarded)
+                let condition = PredicateExpressions.Value(guarded)
                 var pieces: any StandardPredicateExpression<Bool> = PredicateExpressions.build_evaluate(condition, repeat each input)
                 
                 while let next = iterator.next() {

--- a/Sources/FoundationEssentials/Predicate/Predicate.swift
+++ b/Sources/FoundationEssentials/Predicate/Predicate.swift
@@ -56,7 +56,8 @@ extension Predicate {
         
         if let guarded = iterator.next() {
             self.init({ (input: repeat PredicateExpressions.Variable<each Input>) in
-                var pieces: any StandardPredicateExpression<Bool> = PredicateExpressions.build_evaluate(PredicateExpressions.build_Arg(guarded), repeat each input)
+                let condition: PredicateExpressions.Value<Self> = PredicateExpressions.build_Arg(guarded)
+                var pieces: any StandardPredicateExpression<Bool> = PredicateExpressions.build_evaluate(condition, repeat each input)
                 
                 while let next = iterator.next() {
                     pieces = Self.meet(PredicateExpressions.build_evaluate(PredicateExpressions.build_Arg(next), repeat each input), pieces)
@@ -74,7 +75,8 @@ extension Predicate {
         
         if let guarded = iterator.next() {
             self.init({ (input: repeat PredicateExpressions.Variable<each Input>) in
-                var pieces: any StandardPredicateExpression<Bool> = PredicateExpressions.build_evaluate(PredicateExpressions.build_Arg(guarded), repeat each input)
+                let condition: PredicateExpressions.Value<Self> = PredicateExpressions.build_Arg(guarded)
+                var pieces: any StandardPredicateExpression<Bool> = PredicateExpressions.build_evaluate(condition, repeat each input)
                 
                 while let next = iterator.next() {
                     pieces = Self.join(PredicateExpressions.build_evaluate(PredicateExpressions.build_Arg(next), repeat each input), pieces)

--- a/Sources/FoundationEssentials/Predicate/Predicate.swift
+++ b/Sources/FoundationEssentials/Predicate/Predicate.swift
@@ -49,6 +49,7 @@ extension Predicate {
     }
 }
 
+#if FOUNDATION_FRAMEWORK
 @available(FoundationPreview 6.4, *)
 extension Predicate {
     public init(all subpredicates: some BidirectionalCollection<Self>) {
@@ -97,6 +98,7 @@ extension Predicate {
         PredicateExpressions.build_Disjunction(lhs: lhs, rhs: rhs)
     }
 }
+#endif
 
 // Namespace for operator expressions
 @available(macOS 14, iOS 17, tvOS 17, watchOS 10, *)

--- a/Tests/FoundationEssentialsTests/PredicateTests.swift
+++ b/Tests/FoundationEssentialsTests/PredicateTests.swift
@@ -572,6 +572,7 @@ private struct PredicateTests {
         }
     }
     
+    #if FOUNDATION_FRAMEWORK
     @Test func all() throws {
         #expect(try Predicate(all: []).evaluate(42))
         #expect(try !Predicate(all: [Predicate.true, Predicate.false]).evaluate(42))
@@ -661,4 +662,5 @@ private struct PredicateTests {
         #expect(input3.secondAccessed)
         #expect(input3.thirdAccessed)
     }
+    #endif
 }

